static/app/views/relocation/types.ts

```diff
@@ -0,0 +1,17 @@
+import {RouteComponentProps} from 'react-router';
+
+export type StepProps = Pick<
```

Comment:
I'll be honest: I have no idea how to read this type definition. 🤣 But see that we took it from onboarding, so I guess it works here too.